books/free-programming-books-it.md

```diff
@@ -275,6 +275,6 @@
 * [Introduzione alla gestione del progetto software con UML](http://areaprofessional.com/documenti/D03_IntroGestioneProgettoSW.pdf) - G. Destri (PDF)
 
 
-### Visual Basic
+### Visual Basic .NET (VB .Net)
```

Comment:
I'd leave the heading - if you change it you'll need to also change the index. also, we prefer broader categories

Reply:
We like to keep headings consistent across our lists. we use just 'visual basic" in other lists. Put yourself in the shoes of a searcher. -will they type the extra  characters to fine only 'visual basic .net'?

Reply:
Oh yes sorry, didn't know was also cross-list. Done